main/ping.go

```diff
@@ -25,25 +26,35 @@ import (
 	// Using forked version of tarm/serial to force Linux
 	// instead of posix code, allowing for higher baud rates
 	"github.com/uavionix/serial"
+	"github.com/b3nn0/stratux/common"
 )
 
 // Ping device data
 var pingSerialConfig *serial.Config
 var pingSerialPort *serial.Port
 var pingWG *sync.WaitGroup
 var closeCh chan int
+// 0 => pingEFB - 1090ES
+// 1 => pingUSB - MavLink
+var pingDeviceModel int
+var pingDeviceSuccessfullyWorking int
 
 func initPingSerial() bool {
 	var device string
 	baudrate := int(2000000)
 
+	pingDeviceModel = 0
 	log.Printf("Configuring Ping ADS-B\n")
 
 	if _, err := os.Stat("/dev/ping"); err == nil {
 		device = "/dev/ping"
 	} else if _, err := os.Stat("/dev/softrf"); err == nil {
 		device = "/dev/softrf"
 		baudrate = int(38400)
+	} else if _, err := os.Stat("/dev/ttyUSB0"); err == nil {
```

Comment:
This hardcoded path is problematic in many hardware configurations, because other devices might occupy /dev/ttyUSB0, and then the ping will become /dev/ttyUSB1.
To solve this, Stratux uses udev rules (see image/10-stratux.rules).
Please add a rule there that uniquely the ping's properties (vendor/device ID, if not distinct potentially additional attributes) to something like /dev/pingusb or something, then use this path